### ISSUE TITLE
Title screen not drawn properly

### ISSUE THREAD

**Issue by GG-com2016** [ppy/osu#3828]
So somehow after some pretty bad lag while starting the game the title screen is weird. The osu cookie is empty inside. And when i clicked on it there is something missing? I guess because I feel something is not right.

Screenshots:
![capture](https://user-images.githubusercontent.com/24825803/49589647-e06d1c80-f9a4-11e8-9127-4a850bbf3378.PNG)
![image](https://user-images.githubusercontent.com/24825803/49589674-f37fec80-f9a4-11e8-83f6-fe9dd6344b7d.png)

Version: 2018.1203.0
You can see it at the bottom of the screenshot tho.

Log:(This is runtime.log)
```2018-12-06T13:20:21: GL Initialized
2018-12-06T13:20:21: GL Version:                 3.1.0 - Build 9.17.10.4459
2018-12-06T13:20:21: GL Renderer:                Intel(R) HD Graphics 3000
2018-12-06T13:20:21: GL Shader Language version: 1.40 - Intel Build 9.17.10.4459
2018-12-06T13:20:21: GL Vendor:                  Intel
2018-12-06T13:20:21: GL Extensions:              GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_blend_color GL_EXT_abgr GL_EXT_texture3D GL_EXT_clip_volume_hint GL_EXT_compiled_vertex_array GL_SGIS_texture_edge_clamp GL_SGIS_generate_mipmap GL_EXT_draw_range_elements GL_SGIS_texture_lod GL_EXT_rescale_normal GL_EXT_packed_pixels GL_EXT_texture_edge_clamp GL_EXT_separate_specular_color GL_ARB_multitexture GL_EXT_texture_env_combine GL_EXT_bgra GL_EXT_blend_func_separate GL_EXT_secondary_color GL_EXT_fog_coord GL_EXT_texture_env_add GL_ARB_texture_cube_map GL_ARB_transpose_matrix GL_ARB_texture_env_add GL_IBM_texture_mirrored_repeat GL_EXT_multi_draw_arrays GL_NV_blend_square GL_ARB_texture_compression GL_3DFX_texture_compression_FXT1 GL_EXT_texture_filter_anisotropic GL_ARB_texture_border_clamp GL_ARB_point_parameters GL_ARB_texture_env_combine GL_ARB_texture_env_dot3 GL_ARB_texture_env_crossbar GL_EXT_texture_compression_s3tc GL_ARB_shadow GL_ARB_window_pos GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_ARB_vertex_program GL_EXT_texture_rectangle GL_ARB_fragment_program GL_EXT_stencil_two_side GL_ATI_separate_stencil GL_ARB_vertex_buffer_object GL_EXT_texture_lod_bias GL_ARB_occlusion_query GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_shading_language_100 GL_ARB_texture_non_power_of_two GL_ARB_vertex_shader GL_NV_texgen_reflection GL_ARB_point_sprite GL_ARB_fragment_program_shadow GL_EXT_blend_equation_separate GL_ARB_depth_texture GL_ARB_texture_rectangle GL_ARB_draw_buffers GL_ARB_color_buffer_float GL_ARB_half_float_pixel GL_ARB_texture_float GL_ARB_pixel_buffer_object GL_EXT_framebuffer_object GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_occlusion_query2 GL_EXT_draw_buffers2 GL_WIN_swap_hint GL_EXT_texture_sRGB GL_ARB_multisample GL_EXT_packed_float GL_EXT_texture_shared_exponent GL_ARB_texture_rg GL_ARB_texture_compression_rgtc GL_NV_conditional_render GL_EXT_texture_swizzle GL_ARB_sync GL_ARB_framebuffer_sRGB GL_EXT_packed_depth_stencil GL_ARB_depth_buffer_float GL_EXT_transform_feedback GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_ARB_framebuffer_object GL_EXT_texture_array GL_EXT_texture_integer GL_ARB_map_buffer_range GL_EXT_texture_snorm GL_INTEL_performance_queries GL_ARB_copy_buffer GL_ARB_sampler_objects GL_NV_primitive_restart GL_ARB_seamless_cube_map GL_ARB_uniform_buffer_object GL_ARB_depth_clamp GL_ARB_vertex_array_bgra GL_ARB_shader_bit_encoding GL_ARB_draw_buffers_blend GL_ARB_texture_query_lod GL_ARB_explicit_attrib_location GL_ARB_draw_elements_base_vertex GL_ARB_instanced_arrays GL_ARB_fragment_coord_conventions GL_EXT_gpu_program_parameters GL_ARB_texture_buffer_object_rgb32 GL_ARB_compatibility GL_ARB_texture_rgb10_a2ui GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_timer_query GL_INTEL_map_texture GL_ARB_vertex_array_object GL_ARB_provoking_vertex
2018-12-06T13:20:21: GL Context:                 Index: 6, Color: 32 (8888), Depth: 16, Stencil: 0, Samples: 0, Accum: 64 (16161616), Buffers: 2, Stereo: False```

**Comment by peppy** [comment]
How much RAM do you have on your PC? Full specs would likely be helpful too.

**Comment by GG-com2016** [comment]
4gb

**Comment by Eb3rM5** [comment]
The same happens to me on my old PC.
The driver's version is the same of @GG-com2016, but the graphics card is an Intel Graphics 2000, so I'm not gonna send `runtime.log` because the only thing that changes is the GL Renderer.

These are the specs:

> Processor: Intel Core i3-2120 3.30GHz, 3300MHz, 2 cores
Motherboard: BIOSTAR Group H61MLC
RAM: 4GB
S.O: Windows 10 Pro (build 17134)

But tbh, I think this is a problem with Intel drivers, since Intel HD Graphics 3000/2000 aren't supported by Intel anymore. Also, I already had problems with newer versions of LWJGL on that, where OpenGL wouldn't even recognize the card (the same happened with osu!lazer before the migration to netcore-2.1).

**Comment by Joehuu** [comment]
This can be closed as https://github.com/ppy/osu/issues/5199 is probably the same and has more info.

**Comment by iiSaLMaN** [comment]
> This can be closed as #5199 is probably the same and has more info.

Correct, this type of view can be seen with "Bypass front-to-back" enabled (other than the new text builder glyph bug)
